electron/test/main/SingleSignOnTest.js

```diff
@@ -1,10 +0,0 @@
-const assert = require('assert');
```

Comment:
This one seems to not have been translated to TS

Reply:
Mmmmm. I deleted that file. 🐺